cartography/driftdetect/get_states.py

```diff
@@ -90,7 +92,13 @@ def run_get_states(config):
                 logger.exception(err)
 
 
-def get_query_state(session, query_directory, state_serializer, storage, filename):
+def get_query_state(
+        session: neo4j.Session,
+        query_directory: str,
+        state_serializer: StateSchema,
+        storage,
```

Comment:
Not adding a typehint here because it seems like we're doing this ducktyping thing where we expect the `storage` object to have a write() function and other methods but I don't want to necessarily restrict it to _just_ the existing `storage.FileSystem` object.